logger_test.go

```diff
@@ -236,7 +236,7 @@ func wasLogged(t *testing.T, logReader io.Reader) bool {
 	if err != nil && err != io.EOF {
 		t.Fatalf("cannot read log entry: %v", err)
 	}
-	return !(err == io.EOF || len(b) == 0)
+	return err != io.EOF && len(b) != 0
```

Comment:
We are already explicitly checking for `io.EOF` in the if-check before, so we **know** it is not `io.EOF`.

```suggestion
	return len(b) != 0
```